### ISSUE TITLE
hvc.utils.fetch('repo_name',destination='~/some/path`) raises FileNotFoundError

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/hybrid-vocal-classifier#40]
specifically 
```
/anaconda3/envs/hvc/lib/python3.5/site-packages/hvc/utils/datasets.py in _get_http(url, temp_file_name, initial_size, file_size, verbose_bool)
    281                            verbose_bool=verbose_bool)
    282     chunk_size = 8192  # 2 ** 13
--> 283     with open(temp_file_name, mode) as local_file:
    284         while True:
    285             t0 = time.time()

FileNotFoundError: [Errno 2] No such file or directory: 'home/ildefonso/data/sober.repo1.gy6or6.032512.tar.gz'
```
but `hvc.utils.fetch('gy6or6','/home/ildefonso/data/')` works fine,
need to add code to expand `~` I think.

**Comment by NickleDave** [comment]
You want `os.path.expanduser()`

**Comment by NickleDave** [comment]
Fixed with 1cc0b4dcfeafaf4a43d22be3d716860dd47aac70
